paddlenlp/datasets/cote.py

```diff
@@ -68,31 +98,35 @@ def _read(self, filename, split):
             self.name]['splits'][split]
         with open(filename, 'r', encoding='utf-8') as f:
             for idx, line in enumerate(f):
-                if idx < num_discard_samples:
-                    continue
-                line_stripped = line.strip().split('\t')
-                if not line_stripped:
-                    continue
-                example = [line_stripped[indice] for indice in field_indices]
-                if split == "test":
-                    yield {"tokens": list(example[0])}
-                else:
-                    try:
-                        entity, text = example[0], example[1]
-                        start_idx = text.index(entity)
-                    except:
-                        # drop the dirty data
+                try:
+                    if idx < num_discard_samples:
                         continue
+                    line_stripped = line.strip().split('\t')
+                    if not line_stripped:
+                        continue
+                    example = [line_stripped[indice] for indice in field_indices]
+                    if split == "test":
+                        yield {"tokens": list(example[0])}
+                    else:
+                        try:
```

Comment:
Seems this `try` is used to handle lines who have more than one `\t`. It might be better to use `line.split('\t', 1)`  to process lines instead of dropping it.